icu4j/main/classes/core/src/com/ibm/icu/util/MeasureUnit.java

```diff
@@ -121,147 +121,147 @@
         /**
          * SI prefix: yotta, 10^24.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         YOTTA(24, "yotta", 10),
 
         /**
          * SI prefix: zetta, 10^21.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         ZETTA(21, "zetta", 10),
 
         /**
          * SI prefix: exa, 10^18.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         EXA(18, "exa", 10),
 
         /**
          * SI prefix: peta, 10^15.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         PETA(15, "peta", 10),
 
         /**
          * SI prefix: tera, 10^12.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         TERA(12, "tera", 10),
 
         /**
          * SI prefix: giga, 10^9.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         GIGA(9, "giga", 10),
 
         /**
          * SI prefix: mega, 10^6.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         MEGA(6, "mega", 10),
 
         /**
          * SI prefix: kilo, 10^3.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         KILO(3, "kilo", 10),
 
         /**
          * SI prefix: hecto, 10^2.
          *
-         * @draft ICU 68
+         * @draft ICU 69
          */
         HECTO(2, "hecto", 10),
 
         /**
          * SI prefix: deka, 10^1.
          *
-         * @draft ICU 68
+         * @draft ICU 60
```

Comment:
s/60/69/

Remember you can use `git commit --amend` to keep it to one commit.

Reply:
Fixed these typos. Tnx for reminder about ''--amend".